page_object/toDoListPageObject.ts

```diff
@@ -0,0 +1,48 @@
+import {expect} from '@playwright/test';
+
+const ulrTodolist = 'https://webdriveruniversity.com/To-Do-List/index.html'
+const newTodoPlaceholder = 'Add new todo';
+
+export class ToDoListPageObject{
+    async navigate(page: any){
```

Comment:
We should try to not use any. page is type Page. 
page: Page

Reply:
You can make Page as global variable and assign a value in the constructor if you do this you could remove page from parameters in every methods, but your object from line 48 stop work so you will need create object in tests. I suggest to use describe.test for create global variables and assign them in beforeEach.